src/app/feature/certificate/pages/manage-certificates/manage-certificates.component.html

```
@@ -99,26 +99,26 @@ <h2 class="text-xl font-bold mb-6 text-[var(--c-text-secondary)]">
-											<span class="inline-flex items-center rounded-md px-2 py-1 text-xs font-medium ring-1 ring-inset" 
-												[class.bg-blue-500/10]="cert.data?.['templateStyle'] === 'modern'"
-												[class.text-blue-500]="cert.data?.['templateStyle'] === 'modern'"
-												[class.ring-blue-500/20]="cert.data?.['templateStyle'] === 'modern'"
-												[class.bg-amber-500/10]="cert.data?.['templateStyle'] === 'classic'"
-												[class.text-amber-500]="cert.data?.['templateStyle'] === 'classic'"
-												[class.ring-amber-500/20]="cert.data?.['templateStyle'] === 'classic'"
-												[class.bg-slate-500/10]="cert.data?.['templateStyle'] === 'minimalist'"
-												[class.text-slate-500]="cert.data?.['templateStyle'] === 'minimalist'"
-												[class.ring-slate-500/20]="cert.data?.['templateStyle'] === 'minimalist'">
-												{{ cert.data?.['templateStyle'] || 'default' }}
+											<span class="inline-flex items-center rounded-md px-2 py-1 text-xs font-medium ring-1 ring-inset"
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)